### ISSUE TITLE
Is the result based on dev or test set?

### ISSUE THREAD

**Issue by RoderickGu** [google-research/electra#10]
In your paper, sometimes you use dev, sometimes you use dev&test set. But in this github, you did not mention which dataset are you using. Could you clarify? Or would you update the paper? Thanks

----------------------------------
Besdies, what is the training flops you used in the results in the github? Thanks!

**Comment by clarkkev** [comment]
Good question, I updated the README to be clearer about dev/test. You can find the pre-train FLOPs for the models in Table 8 of the paper (under ELECTRA-1.75M, ELECTRA-Base++, and ELECTRA-Small++). ELECTRA-Small-OWT uses the same compute as ELECTRA-Small, 1.4e18 FLOPs.
